homeassistant/components/meteo_france/const.py

```diff
@@ -141,6 +149,22 @@
         ENTITY_ENABLE: True,
         ENTITY_API_DATA_PATH: "current_forecast:clouds",
     },
+    "condition": {
+        ENTITY_NAME: "Condition",
```

Comment:
I'm wondering if we should find another name for the condition to avoid mixing with the one in the weather entity.
In weather entity we filter to reuse standard options. In the entity created in this PR, you are using the original condition description from the API.
What about `original_condition`?

Reply:
OK I will change that by `original_condition` and `daily_original_condition`